frontend/lib/pages/login-page.tsx

```diff
@@ -64,9 +64,12 @@ const LoginPage = withAppContext(
       props.server.originURL
     );
 
+    // Once we have migrated to the same styling for the tenant platform and
```

Comment:
@shakao This feels a little janky to me, but the alternative seems to be overriding the box class on this page on LA LetterBuilder to have...no box. That seems a little more confusing. Let me know if there's a simpler way, happy to amend this.